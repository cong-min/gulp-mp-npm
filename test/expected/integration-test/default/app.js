"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var index_1 = require("./utils/index");
App({
    onLaunch: function (options) {
        console.log('onLaunch options', options);
    },
    onShow: function () { },
    onHide: function () { },
    onError: function () { },
    onPageNotFound: function () { },
    globalData: {},
    emitter: index_1.mitt(),
});

//# sourceMappingURL=app.js.map
