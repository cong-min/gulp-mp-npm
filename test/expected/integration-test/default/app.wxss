/* app.less **/
@import './miniprogram_npm/weui-miniprogram/weui-wxss/dist/style/weui.wxss';
page {
  display: flex;
  flex-direction: column;
  align-items: stretch;
}
