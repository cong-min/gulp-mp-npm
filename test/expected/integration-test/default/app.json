{
  "pages": [
    "pages/index/index"
  ],
  "window": {
    "backgroundTextStyle": "light",
    "navigationBarBackgroundColor": "#fff",
    "navigationBarTitleText": "Gulp Template",
    "navigationBarTextStyle": "black"
  },
  "style": "v2",
  "sitemapLocation": "sitemap.json"
}